### PR TITLE
Contracts: Improve emitted events

### PR DESCRIPTION
**Pull request by facuspagnuolo**
After working on the court subgraph I noticed there are some events we are triggering that are a bit loose and some others that we simply forgot to include certain data 🤷‍♂ 

I needed to adjust the `DisputesManager` and the `JurorsRegistry` only, I recommend reviewing by commit tho.

### CODE REVIEW

File: contracts/disputes/DisputeManager.sol [hunk 1]
```diff
@@ -242,14 +251,15 @@ contract DisputeManager is ControlledRecoverable, ICRVotingOwner, IDisputeManage
...
+        uint256 roundId = dispute.rounds.length - 1;
+        AdjudicationRound storage round = dispute.rounds[roundId];
         uint64 draftTermId = round.draftTermId;
         require(draftTermId <= currentTermId, ERROR_DRAFT_TERM_NOT_REACHED);
         bytes32 draftTermRandomness = clock.ensureCurrentTermRandomness();
 
         // Draft jurors for the given dispute and reimburse fees
         DraftConfig memory config = _getDraftConfig(draftTermId);
-        bool draftEnded = _draft(_disputeId, round, currentTermId, draftTermRandomness, config);
+        bool draftEnded = _draft(round, _buildDraftParams(_disputeId, roundId, currentTermId, draftTermRandomness, config));
```
Comment (bingen) [inline contracts/disputes/DisputeManager.sol]: Was this due to stack too deep issues?

File: contracts/registry/JurorsRegistry.sol [hunk 2]
```diff
@@ -224,15 +226,21 @@ contract JurorsRegistry is ControlledRecoverable, IJurorsRegistry, ERC900, Appro
     * @param _amount Amount of tokens to be added to the available balance of a juror
     */
     function assignTokens(address _juror, uint256 _amount) external onlyDisputeManager {
-        _updateAvailableBalanceOf(_juror, _amount, true);
+        if (_amount > 0) {
+            _updateAvailableBalanceOf(_juror, _amount, true);
+            emit JurorRewarded(_juror, _amount);
```
Comment (bingen) [inline contracts/registry/JurorsRegistry.sol]: The `Reward` part of the event name looks like out of the scope of this function. We could use something more agnostic like `TokensAssigned`.

File: contracts/registry/JurorsRegistry.sol [hunk 3]
```diff
@@ -224,15 +226,21 @@ contract JurorsRegistry is ControlledRecoverable, IJurorsRegistry, ERC900, Appro
...
 
     /**
     * @notice Burn `@tokenAmount(self.token(), _amount)`
     * @param _amount Amount of tokens to be burned
     */
     function burnTokens(uint256 _amount) external onlyDisputeManager {
-        _updateAvailableBalanceOf(BURN_ACCOUNT, _amount, true);
+        if (_amount > 0) {
+            _updateAvailableBalanceOf(BURN_ACCOUNT, _amount, true);
+            emit BurnedTokens(_amount);
```
Comment (bingen) [inline contracts/registry/JurorsRegistry.sol]: For consistency I would call this one `TokensBurned`.

File: contracts/registry/JurorsRegistry.sol [hunk 4]
```diff
@@ -373,7 +385,7 @@ contract JurorsRegistry is ControlledRecoverable, IJurorsRegistry, ERC900, Appro
             tree.update(juror.id, nextTermId, _amount, false);
         }
 
-        emit JurorTokensCollected(_juror, _amount, nextTermId);
+        emit JurorSlashed(_juror, _amount, nextTermId);
```
Comment (bingen) [inline contracts/registry/JurorsRegistry.sol]: I liked better the original name, as the function is not called `slash` but `collectTokens`.

File: contracts/disputes/DisputeManager.sol (hunk 1 above)
Comment (facuspagnuolo) [reply contracts/disputes/DisputeManager.sol]: Yup :/

**Pull request merged**